src/app/clusters/laundry-washer-controls-server/laundry-washer-controls-server.cpp

```diff
@@ -192,3 +192,53 @@ void MatterLaundryWasherControlsPluginServerInitCallback()
     LaundryWasherControlsServer & laundryWasherControlsServer = LaundryWasherControlsServer::Instance();
     registerAttributeAccessOverride(&laundryWasherControlsServer);
 }
+
+Status MatterLaundryWasherControlsClusterServerPreAttributeChangedCallback(const chip::app::ConcreteAttributePath & attributePath,
+                                                                           EmberAfAttributeType attributeType, uint16_t size,
+                                                                           uint8_t * value)
+{
+    Delegate * delegate = GetDelegate(attributePath.mEndpointId);
+    if (delegate == nullptr)
+    {
+        return Status::Success;
+    }
+    switch (attributePath.mAttributeId)
+    {
+    case Attributes::SpinSpeedCurrent::Id: {
+        if (NumericAttributeTraits<uint8_t>::IsNullValue(*value))
+        {
+            return Status::Success;
+        }
+        char buffer[LaundryWasherControlsServer::kMaxSpinSpeedLength];
+        MutableCharSpan spinSpeed(buffer);
+        auto err = delegate->GetSpinSpeedAtIndex((*value), spinSpeed);
```

Comment:
Suggest a helper variable for `*value* that is called `uint8_t spinSpeedIndex = *value;` or such.

Reply:
Yes, I will change it.